torch/lib/TensorLib/common_with_cwrap.py

```diff
@@ -0,0 +1,111 @@
+# this code should be common among cwrap and TensorLib preprocessing
+# for now, I have put it in one place but right now is copied out of cwrap
```

Comment:
We will get this in a common place before merging this. @killeent do think you can look into where to put these and how to route both libraries to use them?